docs/mdbook/default.nix

```diff
@@ -66,11 +66,16 @@ let
         # Make sure this path has a valid info attrset
         if info ? file && info ? description && info ? url then
           "# ${lib.last path}\n\n"
-          + (lib.optionalString (info.description != null) "${info.description}\n\n")
           + (lib.optionalString (info.url != null) "**URL:** [${info.url}](${info.url})\n\n")
           + (lib.optionalString (
             maintainers != [ ]
           ) "**Maintainers:** ${lib.concatStringsSep ", " maintainersNames}\n\n")
+          + lib.optionalString (info.description != null) ''
+
+            ---
+
+            ${info.description}\n\n
```

Comment:
```suggestion
            ${info.description}

```

Reply:
Alternatively, we could format the whole thing using `""`:

```nix
"\n---\n\n"{info.description}\n\n"
```

Reply:
> Alternatively, we could format the whole thing using `""`:
> 
> ```nix
> "\n---\n\n"{info.description}\n\n"
> ```

Sorry, didn't see this. :S

Reply:
I've noticed GitHub isn't great at updating pages you already have opened. I find myself reloading far too often to check if I've missed anything 😂 

No big deal this time though!